src/Microsoft.AspNetCore.SignalR.Core/HubConnectionContext.cs

```diff
@@ -4,43 +4,52 @@
 using System;
 using System.Collections.Concurrent;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.Runtime.ExceptionServices;
 using System.Security.Claims;
 using System.Threading;
-using System.Threading.Tasks;
 using System.Threading.Channels;
+using System.Threading.Tasks;
 using Microsoft.AspNetCore.Http.Features;
+using Microsoft.AspNetCore.SignalR.Core;
+using Microsoft.AspNetCore.SignalR.Core.Internal;
 using Microsoft.AspNetCore.SignalR.Features;
 using Microsoft.AspNetCore.SignalR.Internal;
+using Microsoft.AspNetCore.SignalR.Internal.Encoders;
 using Microsoft.AspNetCore.SignalR.Internal.Protocol;
 using Microsoft.AspNetCore.Sockets;
 using Microsoft.AspNetCore.Sockets.Features;
+using Microsoft.Extensions.Logging;
 
 namespace Microsoft.AspNetCore.SignalR
 {
     public class HubConnectionContext
```

Comment:
General feedback for this class:
- Group method and properties by accessibility. public props then internal props (same for methods).
- The Input channel shouldn't be public. It would mean that any body can read from the HubConnection context which is fundamentally broken.

Reply:
Agreed, this class has needed some clean-up for a while